guides/office/kitchen.md

```diff
@@ -0,0 +1,17 @@
+# Kitchen Etiquette
+
+We share the kitchen space with each other as well as with our customers. We have put together a few guidelines for keeping the kitchen nice and tidy.
+
+## Coffee
+
+We love our coffee at Made Tech. We have fresh beans regularly delivered to the office and have multiple thermos drip coffee jugs that keep coffee lovely and fresh. Please ensure:
+
+ - When grinding beans that no meetings are going on in the meeting room as it's very loud and can interrupt voice calls and conversations
+ - When you're done making coffee that you tidy up any spillages so the kitchen is left in a clean state
+
+## Sink regime
+
+We all appreciate a clear and useable sink. Please ensure:
+
+ - When you're finished with glasses, cutlery and crockery that you wash them up
```

Comment:
Don't believe this happens at the moment, would mean a lot of washing up happening at the end of the day.  I'd be pro this nonetheless.